### DISCUSSION TITLE
Architecture independence

### DISCUSSION THREAD

**Discussion by 0Grit** [Open-CMSIS-Pack/Open-CMSIS-Pack-Spec#17] [Q&A]
Will there be a concerted effort to make this a universal standard by encouraging uptake even by projects which build for other architectures? (Example: Zephyr)

We need a defacto standard for package and dependency management so badly in embedded. It's possibly the largest bottleneck holding back the IoT market.

**Comment by 0Grit** [comment]
No?

@ReinhardKeil

**Comment by jkrech** [comment]
@0Grit, I do agree that there should be no limits on the supported architectures but this will require contributions from people who have the necessary expertise.
